js/about/adblock.js

```diff
@@ -64,11 +65,11 @@ class AboutAdBlock extends React.Component {
   }
   render () {
     const lastUpdateDate = new Date(this.state.adblock.get('lastCheckDate'))
-    return <div className='adblockDetailsPage'>
-      <h2 data-l10n-id='adblock' />
+    return <div className={css(styles.adblockDetailsPage)}>
+      <h2 className={css(styles.adblockDetailsPage__h2)} data-l10n-id='adblock' />
       <list>
         <div role='listitem'>
-          <div className='adblockDetailsPageContent'>
+          <div className={css(styles.adblockDetailsPageContent)}>
             <div className='adblockCount'><span data-l10n-id='blockedCountLabel' /> <span className='blockedCountTotal'>{this.state.adblock.get('count') || 0}</span></div>
```

Comment:
we can remove `adblockCount` and `blockedCountTotal` classes since they're not used anywhere.

For `blockedCountTotal` there's a test written targeting that selector (see [`adBlockTest.js`](https://github.com/brave/browser-laptop/blob/master/test/about/adblockTest.js#L22)) but we can replace className to `data-test-id='blockedCountTotal'` and on the test file we just update to:

```
.getText('[data-test-id="blockedCountTotal"]').should.eventually.be.equal('0')
```